app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PostsController < ApplicationController
+
+  def index
+    @posts = Post.all
+    @comments = Comment.all
```

Comment:
do you want all of the comments here? how will you know which ones are for which post? In the view when we iterate through posts we can do something with all it's comments:

```ruby
@posts.each do |post|
  @post.comments.each do |comment|
    # does somethings with each comment
  end
end
```